node-api-postgres/sample.js

```diff
@@ -0,0 +1,22 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'LMS',
+  password: '123456789',
+  port: 5432,
+})
+
+
+function getUsers() {
+    pool.query('SELECT * FROM public.users', (error, results) => {
+        if(error) {
+            throw error;
+        }
+        for (let user of results.rows){
+            console.log(user);
+        }
+    })
+}
+
+getUsers();
```

Comment:
Should remove this sample.js file since it doesn't relate to the API code

Reply:
ok